app/assets/stylesheets/system_satisfaction.sass

```diff
@@ -36,15 +36,29 @@
     .title
       display: none
     ol
-      padding: 0 8px
+      padding: 0 !important
```

Comment:
Is it possible to do this with specificity, instead of an !important declaration? Generally speaking, important should be used for debugging, and it would be good to cut down on our use of it in production code.